src/utils/container-runtime/src/container.rs

```diff
@@ -51,20 +51,17 @@ impl ContainerRunCommand {
         self
     }
 
-    pub fn container_name_prefix(mut self, prefix: impl AsRef<str>) -> Self {
-        use rand::distributions::Alphanumeric;
-        use rand::Rng;
-
-        let mut name = String::with_capacity(10 + prefix.as_ref().len());
-        name.push_str(prefix.as_ref());
-        name.extend(
-            rand::thread_rng()
-                .sample_iter(&Alphanumeric)
-                .take(10)
-                .map(char::from),
-        );
+    pub fn random_container_network_with_prefix(mut self, prefix: impl AsRef<str>) -> Self {
```

Comment:
I don't think running a container auto-creates a network:
```sh
docker run -it --rm --network xxx ghcr.io/kamu-data/kamu-base:latest-with-data
docker: Error response from daemon: network xxx not found.
```
Thus I don't see how this method can work.
Something needs to create a network with random name first and then pass its name into containers.

Reply:
this method is equal to `network` method which sets network parameters (but in this case with randomization). I'll create on for creation as well